Agents/AccessAgent/README.md

```diff
@@ -1,163 +1,152 @@
-# Introduction
+# Access Agent
 
-The purpose of the Access Agent is to provide a point of access to the knowledge graph for executing SPARQL query and update operations. The Access Agent hides the storage implementation layer, so that the user or calling agent does not need to know the store location or software e.g. Blazegraph, RDF4J etc.
+The purpose of the Access Agent is to provide a point of access to the knowledge graph for executing SPARQL operations and connecting to the associated relational database. The Access Agent hides the storage implementation layer, so that the user or calling agent does not need to know the store location or software e.g. Blazegraph, RDF4J etc. It consists of two sub-agents:
 
-The Access Agent works by receiving a HTTP request from the calling agent to execute a SPARQL query or update on a target resource (triple store). A StoreRouter and the associated *ontokgrouter* triple store is used to retrieve the SPARQL endpoints for the requested resource. A StoreClient is then instantiated to connect to and query or update the target resource.
+**Knowledge Graph Access Agent**: Handles HTTP requests to perform SPARQL query and update operations on RDF resources in the knowledge graph as well as to "get" and "insert" entire graphs. As an extension of The World Avatar's agent framework, this agent's methods can be called **EITHER** using the `AccessAgentCaller` class in the `jps_base_lib` package **OR** by extending the JPSAgent class.
 
-## Purpose
+**RDB Access Agent**: Handles HTTP requests to obtain the url required to create a connection to a relational database. As an extension of The World Avatar's agent framework, this agent's methods can be called **EITHER** using the `RDBAccessAgentCaller` class in the `jps_base_lib` package **OR** by extending the JPSAgent class.
 
-The purpose of the TripleStoreAccessAgent and RDBAccessAgent is to provide access and/or routing to triple stores and relational databases respectively.
```

Comment:
Apart from the "separate agent" language, this statement about the purpose is something I think we should keep.

Reply:
This has been edited and added back